mix.exs

```diff
@@ -15,7 +15,10 @@ defmodule ReduxablePlug.Mixfile do
   # Type "mix help compile.app" for more information
   def application do
     # Specify extra applications you'll use from Erlang/Elixir
-    [extra_applications: [:logger]]
+    [
+      extra_applications: [:logger],
+      mod: {ReduxablePlug.App, []}
```

Comment:
It looks for a `ReduxablePlug.App` module and calls it's `start` function with zero arguments (`[]`)